media/css/firefox/channel.scss

```diff
@@ -44,6 +44,11 @@ $image-path: '/media/protocol/img';
     }
 }
 
+
+.os_linux64 { /* stylelint-disable-line */
+    margin-bottom: $spacing-md;
+}
```

Comment:
Might it be worth trying to keep download button specific selectors in `_download-button.scss`, or would it cause issues elsewhere?

Reply:
I see this issue on the `/developer/` page too - and do we need to add the debian message there as well?

http://localhost:8000/en-US/firefox/developer/

![image](https://github.com/mozilla/bedrock/assets/400117/905577d1-512c-407d-847d-125d2a4d9fc7)